pkg/columns/filter/filter_test.go

```diff
@@ -0,0 +1,285 @@
+		{filterString: "int64:!1", expectedCount: 4, expectError: false},   // match on int64, negated
+		{filterString: "int64:<2", expectedCount: 1, expectError: false},   // match on int64, lt
+		{filterString: "int64:<=1", expectedCount: 1, expectError: false},  // match on int64, lte
+		{filterString: "int64:>1", expectedCount: 4, expectError: false},   // match on int64, gt
+		{filterString: "int64:>=1", expectedCount: 5, expectError: false},  // match on int64, gte
+		{filterString: "uint64:", expectedCount: 0, expectError: true},     // match on uint64, empty string
+		{filterString: "uint64:1", expectedCount: 1, expectError: false},   // match on uint64, empty string
+		{filterString: "uint64:~1", expectedCount: 0, expectError: true},   // match on uint64, wrong comparison type
+		{filterString: "uint64:!1", expectedCount: 4, expectError: false},  // match on uint64, negated
+		{filterString: "uint64:<2", expectedCount: 1, expectError: false},  // match on uint64, lt
+		{filterString: "uint64:<=1", expectedCount: 1, expectError: false}, // match on uint64, lte
+		{filterString: "uint64:>1", expectedCount: 4, expectError: false},  // match on uint64, gt
+		{filterString: "uint64:>=1", expectedCount: 5, expectError: false}, // match on uint64, gte
+
+		{filterString: "float32:", expectedCount: 0, expectError: true},     // match on float32, empty string
+		{filterString: "float32:1", expectedCount: 1, expectError: false},   // match on float32, empty string
+		{filterString: "float32:~1", expectedCount: 0, expectError: true},   // match on float32, wrong comparison type
+		{filterString: "float32:!1", expectedCount: 4, expectError: false},  // match on float32, negated
+		{filterString: "float32:<2", expectedCount: 1, expectError: false},  // match on float32, lt
+		{filterString: "float32:<=1", expectedCount: 1, expectError: false}, // match on float32, lte
+		{filterString: "float32:>1", expectedCount: 4, expectError: false},  // match on float32, gt
+		{filterString: "float32:>=1", expectedCount: 5, expectError: false}, // match on float32, gte
+		{filterString: "float64:", expectedCount: 0, expectError: true},     // match on float64, empty string
+		{filterString: "float64:1", expectedCount: 1, expectError: false},   // match on float64, empty string
+		{filterString: "float64:~1", expectedCount: 0, expectError: true},   // match on float64, wrong comparison type
+		{filterString: "float64:!1", expectedCount: 4, expectError: false},  // match on float64, negated
+		{filterString: "float64:<2", expectedCount: 1, expectError: false},  // match on float64, lt
+		{filterString: "float64:<=1", expectedCount: 1, expectError: false}, // match on float64, lte
+		{filterString: "float64:>1", expectedCount: 4, expectError: false},  // match on float64, gt
+		{filterString: "float64:>=1", expectedCount: 5, expectError: false}, // match on float64, gte
```

Comment:
All these comments could become the test name to pass to `t.Run()` below. (same comment applies to other parts on the code).

Reply:
Moved from comment to field for a couple of tests now => tests much more readable.